docs/services/postgresql/add-standby-node.md

```diff
@@ -32,6 +32,84 @@ $ cchq <env> ap setup_pg_standby.yml -e standby=[standby node]
 
 
 # Promoting a hot standby to Master
+1. Begin downtime for your site: `$ commcare-cloud <env> downtime start`
+
+2. Verify that the replication is up to date
+
+```
+$ commcare-cloud <env> run-shell-command pg1,pg2 'ps -ef | grep -E "sender|receiver"'
+
+    [ pg1 ] ps -ef | grep -E "sender|receiver"
+    postgres 5295 4517 0 Jul24 ? 00:00:01 postgres: wal sender process rep 10.116.175.107(49770) streaming 0/205B598
+
+    [ pg2 ] ps -ef | grep -E "sender|receiver"
+    postgres 3821 3808 0 Jul24 ? 00:01:27 postgres: wal receiver process streaming 0/205B598
+```
+
+Output shows that master and standby are up to date (both processing the same log).
+
+3. Promote the standby
+
 ```
-$ cchq <env> ansible-paybook promote_pg_standby.yml -e standy=[standby nde]
+$ commcare-cloud <env> ansible-paybook promote_pg_standby.yml -e standy=[standby node]
+```
+
+4. In your inventory remove `hot_standby_master` and `replication_slot` variables from your standby node, and remove the node from the `pg_standby` group.
+
+5. Update your processes to point to the newly promoted server:
+
+```
+$ commcare-cloud <env> update-config
+```
+
+6. If the standby you've promoted is one of the `form_processing` databases, update the PL proxy cluster
```

Comment:
This would require editing the `postgresql.yml` config file right?

Reply:
https://github.com/dimagi/commcare-cloud/pull/3072/commits/8194aaa7f75c64e86bea8644152a526ec6491712#diff-f05f8aa174fde4064a27d369a976954fR74 (accidentally put it in an unrelated commit)